Remove duplicated/outdated documentation that is already inside ClickHouse (#7944)
fishtown-analytics/docs.getdbt.com: website/docs/docs/core/connect-data-platform/clickhouse-setup.md

<SetUpPages meta={frontMatter.meta} />

## Supported features

- [x] Table materialization
- [x] View materialization
- [x] Incremental materialization
- [x] Microbatch incremental materialization

> - [x] Materialized View materializations (uses the `TO` form of MATERIALIZED VIEW, experimental)

Reviewer: ```suggestion
- [x] Materialized View materialization (experimental)
```

- [x] Seeds
- [x] Sources
- [x] Docs generate
- [x] Tests
- [x] Snapshots
- [x] Most dbt-utils macros (now included in dbt-core)

> - [x] Ephemeral materialization

Reviewer: Could we bring all the materialization types together? As a table that looks more like [this](https://docs.getdbt.com/docs/core/connect-data-platform/materialize-setup#supported-features)? We might not even need it, since we don't really deviate from the base materializations in any way (maybe save for materialized views).

Author: I have tried to change it. Let me know what you think :D 

<img width="1026" height="1232" alt="Screenshot 2025-10-15 at 18 50 55" src="https://github.com/user-attachments/assets/cac6f421-97a9-46df-8831-063b1d83a151" />

- [x] Distributed table materialization (experimental)
- [x] Distributed incremental materialization (experimental)
- [x] Contracts

> ## Connecting to ClickHouse with **dbt-clickhouse**

Reviewer: ```suggestion
## Connecting to ClickHouse
```

Reviewer: For consistency with other pages. This section should come first, too!

To connect to ClickHouse from dbt, you'll need to add a [profile](/docs/core/connect-data-platform/connection-profiles)

> to your `profiles.yml` file or inside your `~/.dbt/profiles.yml` file. An example of a ClickHouse profile has the following syntax:

Reviewer: ```suggestion
to your `profiles.yml` configuration file. Follow the reference configuration below to set up a ClickHouse profile:
```

<File name='profiles.yml'>
```yaml
clickhouse-service:
  target: dev
  outputs:
    dev:
      type: clickhouse
      schema: [ default ] # ClickHouse database for dbt models

      # optional

>       host: [ localhost ]

Reviewer: Let's default to a cloud host (or a hint to CH Cloud), rather than local.

Author: Done! Let me know what you think :D

      port: [ 8123 ]  # Defaults to 8123, 8443, 9000, 9440 depending on the secure and driver settings 
      user: [ default ] # User for all database operations
      password: [ <empty string> ] # Password for the user
      secure: [ False ] # Use TLS (native protocol) or HTTPS (http protocol)

      # You can find all the configurations options in the ClickHouse documentation: https://clickhouse.com/docs/integrations/dbt

> ```

Reviewer: ````suggestion
```

For a complete list of configuration options, see the [ClickHouse documentation](https://clickhouse.com/docs/integrations/dbt).
````

</File>

### Create a dbt project
You can now use this profile in one of your existing projects or create a new one using:

```sh
dbt init project_name
```

Inside `project_name` dir, update your `dbt_project.yml` file to specify a profile name to connect to the ClickHouse server.

```yaml
profile: 'clickhouse-service'
```

### Test connection
Execute `dbt debug` with the CLI tool to confirm whether dbt is able to connect to ClickHouse. Confirm the response includes `Connection test: [OK connection ok]` indicating a successful connection.

## Documentation

See the [ClickHouse website](https://clickhouse.com/docs/integrations/dbt) for the full documentation entry for the `dbt-clickhouse` adapter.

## Contributing

We welcome contributions from the community to help improve the dbt-ClickHouse adapter. Whether you're fixing a bug,

> adding a new feature, or enhancing the documentation, your efforts are greatly appreciated!

Reviewer: ````suggestion
Navigate to the `project_name` directory and update your `dbt_project.yml` file to use the profile you configured to connect to ClickHouse.

```yaml
profile: 'clickhouse-service'
```

### Test connection

Execute `dbt debug` with the CLI tool to confirm whether dbt is able to connect to ClickHouse. Confirm the response includes `Connection test: [OK connection ok]`, indicating a successful connection.

## Documentation

See the [ClickHouse documentation](https://clickhouse.com/docs/integrations/dbt) for more details on using the `dbt-clickhouse` adapter to manage your data model.

## Contributing

We welcome contributions from the community to help improve the `dbt-ClickHouse` adapter. Whether you're fixing a bug,
adding a new feature, or enhancing the documentation, your efforts are greatly appreciated!
````

Author: Sorry, did you only want to change the last phrase or did you also wanted to remove all these sections?

Reviewer: Freaking backticks. Just patched the suggestion!

View | YES | Creates a [view](https://clickhouse.com/docs/en/sql-reference/table-functions/view/).
Incremental | YES | Creates a table if it doesn't exist, and then writes only updates to it.
Microbatch incremental | YES | 
Ephemeral materialization | YES | Creates a ephemeral/CTE materialization. This model is internal to dbt and does not create any database objects.
Materialized View | YES, Experimental | Creates a [materialized view](https://clickhouse.com/docs/en/materialized-view).
Distributed table materialization | YES, Experimental | Creates a [distributed table](https://clickhouse.com/docs/en/engines/table-engines/special/distributed).
Distributed incremental materialization | YES, Experimental | Incremental model based on the same idea as distributed table. Note that not all strategies are supported, visit [this](https://github.com/ClickHouse/dbt-clickhouse?tab=readme-ov-file#distributed-incremental-materialization) for more info.
Dictionary materialization | YES, Experimental | Creates a [dictionary](https://clickhouse.com/docs/en/engines/table-engines/special/dictionary).

> **Note**: Community-developed features are labeled as experimental. Despite this designation, many of these features, like materialized views, are widely adopted and successfully used in production environments.

Reviewer: Nice!
